File: CleanJavaRx/app/src/main/java/com/vascome/fogtail/screens/main/fragment/list/adapter/VerticalSpaceItemDecoration.java
package com.vascome.fogtail.screens.main.fragment.list.adapter;

import android.graphics.Rect;
import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.View;

/**
 * Created by vasilypopov on 12/6/17
 * Copyright (c) 2017 MVPJava. All rights reserved.
 */

public class VerticalSpaceItemDecoration extends RecyclerView.ItemDecoration {

    private final int verticalSpacePx;

    public VerticalSpaceItemDecoration(int verticalSpacePx) {
        this.verticalSpacePx = verticalSpacePx;
    }

    @Override
    public void getItemOffsets(@NonNull Rect outRect, @NonNull View view, @NonNull RecyclerView parent, @NonNull RecyclerView.State state) {
        outRect.top = verticalSpacePx;
        outRect.bottom = verticalSpacePx;
    }
}

File: MosbyMVPConductorKotlin/app/src/main/java/com/vascome/fogtail/presentation/main/fragment/list/adapter/VerticalSpaceItemDecoration.kt
package com.vascome.fogtail.presentation.main.fragment.list.adapter

import android.graphics.Rect
import android.support.v7.widget.RecyclerView
import android.view.View

/**
 * Created by vasilypopov on 12/6/17
 * Copyright (c) 2017 MVPJava. All rights reserved.
 */

class VerticalSpaceItemDecoration(private val verticalSpacePx: Int) : RecyclerView.ItemDecoration() {

    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
        outRect.top = verticalSpacePx
        outRect.bottom = verticalSpacePx
    }
}